code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -208,3 +208,40 @@ Difficulty: Medium
 	playsound(user, 'sound/magic/staff_change.ogg', 200, 0)
 	A.telegraph()
 	storm_cooldown = world.time + 200
+
+//Crown and Jewel of the Wastes
+/obj/item/clothing/ears/jewel_of_the_wastes //Takes headset but makes you have ear protection + same faction as the simple mobs
+	name = "jewel of the wastes"
+	desc = "A large blood red gem with metal clasps on it side. An echoing scream can be heard if placed next to the ear"
+	w_class = WEIGHT_CLASS_TINY
+	slot_flags = ITEM_SLOT_EARS
+	icon_state = "jewel_of_wastes"
+	item_state = "jewel_of_wastes"
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+
+/obj/item/clothing/ears/jewel_of_the_wastes/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(SLOT_EARS))
+
+/obj/item/clothing/ears/jewel_of_the_wastes/equipped(mob/user, slot)
+	..()
+	user.faction |= "mining"
```

Comment:
Doesn't check which slot they are equipped to and also lacks dropped() proc, which means it will permanently grant anyone the mining faction as soon as it's picked up.
And even if it does happen to have a `dropped()` faction removal line, it will end up potentially removing said faction from any theorical not-handless mob that previously had it.